tests/integration/data_finder.yml

```diff
@@ -42,6 +44,10 @@ get_input_filelist:
       - ta_Amon_HadGEM2-ES_historical_r1i1p1*.nc
     found_files:
       - ta_Amon_HadGEM2-ES_historical_r1i1p1_195912-198411.nc
+    found_timerange_files:
```

Comment:
`get_input_filelist` should find the same files, regardless of whether a timerange or the same information with a start_year and end_year is provided. Therefore I feel that this new organization of the tests is confusing. From the associated test, it looks like this section is intended to test that using a '*' for the timerange works, right? I think it would be better to just add some new testcases to this file for that purpose, without modifying the test itself. For example
```yaml
get_input_filelist:
  - drs: default
    variable:
      variable_group: test
      short_name: ta
      original_short_name: ta
      dataset: HadGEM2-ES
      project: CMIP5
      cmor_table: CMIP5
      institute: [INPE, MOHC]
      frequency: mon
      modeling_realm: [atmos]
      mip: Amon
      exp: historical
      ensemble: r1i1p1
      timerange: '*'
      diagnostic: test_diag
      preprocessor: test_preproc
    available_files:
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_193412-195911.nc
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_195912-198411.nc
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_198412-200511.nc
    dirs:
      - ''
    file_patterns:
      - ta_Amon_HadGEM2-ES_historical_r1i1p1*.nc
    found_files:
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_193412-195911.nc
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_195912-198411.nc
      - ta_Amon_HadGEM2-ES_historical_r1i1p1_198412-200511.nc
 ```